backend/fms_core/containers.py

```diff
@@ -145,6 +146,14 @@ def __str__(self):
 
 # Run Containers
 
+CONTAINER_SPEC_AXIOM_ARRAY = ContainerSpec(
+    container_kind_id="axiom genetitan array",
```

Comment:
"axiom 96-format array" instead. There might not be a one-to-one association to genetitan and there is a 384 array format available (we might get to add them at some point).